AddonBuilder/Classes/Updates/GetLatestVersion.cs

```diff
@@ -8,7 +7,7 @@ class GetLatestVersion
     {
         class GitHubData
         {
-            public string tag_name { get; set; }
+            public string TagName { get; set; }
```

Comment:
I know this isn't supposed how to be written but it gets the data from a JSON and the serializer might need the exact name. Did you test if it returns the correct version number?

Reply:
That's just a property that encapsulates a variable. All properties are pascal case in terms of naming conventions similar to methods as they essentially implement getters and setters for that corresponding field. Now if your asking if the renaming would mess up the code, the renamer in resharper is smart of enough to detect all usages in the solution and update them accordingly.